images/cluster-setup/install.sh

```diff
@@ -20,122 +20,127 @@ set -o pipefail
 
 usage() {
 
-    printf "Usage: DATA_DIR="/workspace" ./register.sh \n\n"
-
-    # Parameters
-    printf "The following parameters need to be passed to the script: \n"
-    printf "DATA_DIR: the location of the cluster files \n"
+  printf "
+Usage:
```

Comment:
As explained in the other PR environment variables are a more convenient approach.

Reply:
I did not remove support for env variable. I just added support for arguments on top.